Read fix (#5)
tmadlener/k4EDM4hep2LcioConv: standalone/README.md

> To convert an lcio file to edm4HEP either a lcio file where all collections are present in every event

Reviewer: ```suggestion
To convert an lcio file to EDM4hep either a LCIO file where all collections are present in every event
```

or a file containing a list of all collections in the file are needed. 

> This list can be obtained by executing check_missing_cols from the lcio package with the --minimal flag.

Reviewer: ```suggestion
This list can be obtained by executing `check_missing_cols` from the LCIO package with the `--minimal` flag.
```

Example:

> ./lcio/install/bin/check_missing_cols /pnfs/desy.de/ilc/prod/ilc/mc-2020/ild/rec/250-SetA/higgs/ILD_l5_o2_v02/v02-02-01/00015671/000/rv02-02-01.sv02-02-01.mILD_l5_o2_v02.E250-SetA.I402005.Pe3e3h.eL.pR.n000_002.d_rec_00015671_493.slcio > patch.txt
> 

Reviewer: ```suggestion
```bash
check_missing_cols --minimal \
  /pnfs/desy.de/ilc/prod/ilc/mc-2020/ild/rec/250-SetA/higgs/ILD_l5_o2_v02/v02-02-01/00015671/000/rv02-02-01.sv02-02-01.mILD_l5_o2_v02.E250-SetA.I402005.Pe3e3h.eL.pR.n000_002.d_rec_00015671_493.slcio \
  > patch.txt
```
```

You can place these into a `bash` markdown block for slightly nicer viewing

This file will be the third argument when the converter ist called. 

> ./standalone/lcio2edm4hep /pnfs/desy.de/ilc/prod/ilc/mc-2020/ild/rec/250-SetA/higgs/ILD_l5_o2_v02/v02-02-01/00015671/000/rv02-02-01.sv02-02-01.mILD_l5_o2_v02.E250-SetA.I402005.Pe3e3h.eL.pR.n000_002.d_rec_00015671_493.slcio Output.root patch.txt

Reviewer: ```suggestion
```bash
lcio2edm4hep \
  /pnfs/desy.de/ilc/prod/ilc/mc-2020/ild/rec/250-SetA/higgs/ILD_l5_o2_v02/v02-02-01/00015671/000/rv02-02-01.sv02-02-01.mILD_l5_o2_v02.E250-SetA.I402005.Pe3e3h.eL.pR.n000_002.d_rec_00015671_493.slcio \
  Output.root \
  patch.txt
```
```

same here

The resulting root file will be "Output.root" 

A few things to note:

> ralations that do not have a "toType" and "fromType" set are currently not beeing converted. If check_missing_cols is run without the minimal flag a warning will be given for each Ralations this applies to.

Reviewer: ```suggestion
- LCRelations that do not have a `"ToType"` and `"FromType"` as collection parameters set are currently not being converted. If `check_missing_cols` is run without the `--minimal` flag a warning will be given for each Relation this applies to.
```
Fix some types and also make this a list

To convert aa LCIO file to EDM4hep either a LCIO file where all collections are present in every event
or a file containing a list of all collections in the file are needed. 
This list can be obtained by executing 'check_missing_cols' from the LCIO package with the '--minimal' flag.

> 

Reviewer: Can you put a small sentence and an example for the format of these files here as well? So that people know how it should look like if they want to create one from scratch.

Also add another sentence, that this file is simply used to patch missing collections and cannot be used to select which collections will be converted.
